CatoNetworks/cato-sase/ingest/parser.yml

```
@@ -0,0 +1,85 @@
+name: cato_sase
+pipeline:
+  - name: json_event
+    external:
+      name: json.parse-json
+      properties:
+        input_field: "{{original.message}}"
+        output_field: output
+
+  - name: set_common_fields
+
+stages:
+  set_common_fields:
+    actions:
+      # Default fields with predefined values
+      - set:
+          event.kind: "event"
+          event.action: "{{json_event.output.action}}"
```

I suggest lowering the case of the action.
```suggestion
          event.action: "{{json_event.output.action | lower}}"
```